app/assets/stylesheets/argo/argo.sass

```diff
@@ -131,8 +131,10 @@ table
 
 .bulk_button
   cursor: pointer
-  width: 100px
-  padding: 4px 6px
+  width: 160px
```

Comment:
By using the Bootstrap `btn` and `btn-lg` classes you could probably get rid of these classes